cmd/cluster/kubevirt/create.go

```diff
@@ -0,0 +1,76 @@
+package kubevirt
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	"github.com/spf13/cobra"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "kubevirt",
+		Short:        "Creates basic functional HostedCluster resources for KubeVirt platform",
+		SilenceUsage: true,
+	}
+
+	opts.KubevirtPlatform = core.KubevirtPlatformCreateOptions{
+		Memory: "4Gi",
+		CPU:    2,
+		Image:  "",
+	}
+
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.Memory, "kubevirt-memory", opts.KubevirtPlatform.Memory, "In KubeVirt platform - the amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
+	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.CPU, "kubevirt-cpu", opts.KubevirtPlatform.CPU, "In KubeVirt platform - Cores specifies the number of cores inside the vmi, Must be a value greater or equal 1")
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.Image, "kubevirt-image", opts.KubevirtPlatform.Image, "In KubeVirt platform - Image is the name of the image with the embedded disk to be used to create the machines")
```

Comment:
how about `containerdisk`, `rootdisk`, `node-containerdisk` with a descriptin like: 

```
OpenShift compatible kubevirt containerdisk with ignition support (e.g. RHCOS, FCOS) 
```

Reply:
Changed it to containerdisk